materials/section-3/README.md

```diff
@@ -11,17 +11,99 @@
 
 ## 1. Introduction to ClojureScript
 
-TBD:
-- differences between clj and cljs
-- js interop
+### What is ClojureScript?
+
+Taken directly from the ClojureScript [website](https://clojurescript.org):
+
+> ClojureScript is a compiler for Clojure that targets JavaScript. It emits JavaScript code which is compatible with the advanced compilation mode of the Google Closure optimizing compiler.
+
+The Google Closure compiler allows for your ClojureScript code to be compiled into highly optimised JavaScript code. It (in it's own words) "compiles from JavaScript to better JavaScript".
+
+
+### So, why not just use JavaScript?
+
+I think we can all agree here that JavaScript has _some_ shortcomings. Since JavaScript is quite a feature-rich language, there are many corners in which to hide bugs in your code. It requires extensive discipline from the JavaScript developers to ensure that the software written is bug-free. The amount of effort required to write robust code in JavaScript is vastly higher than to achieve the same feat in ClojureScript.
```

Comment:
This is actually a really tricky question to answer aside from (ClojureScript is nicer than JavaScript, _duh_). Do you have any thoughts on what else could be said here @ykarikos ?